From ebcdff48c8efbcb0044e0ef39bc3c3d41f0bb9fe Mon Sep 17 00:00:00 2001
From: Varsha Maira <vmaira@habana.ai>
Date: Wed, 28 May 2025 17:51:52 +0300
Subject: [PATCH] [SW-230471] g3: mgmt: disable assert to reduce iccm usage

With latest code most of the ICCM is utilized and unable to add new
features like secure protection bits. In order to reduce ICCM size
ASSERT is disabled

Change-Id: I31dca4f7556e4ca4ec2e21c4cec023a322633e15
Signed-off-by: Varsha Maira <vmaira@habana.ai>
---

diff --git a/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig b/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
index 30d6738..a5fd7d5 100644
--- a/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
+++ b/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
@@ -164,6 +164,6 @@
 CONFIG_LFS_LOG_FILE_SIZE=32768
 CONFIG_LFS_LOG_FILES_LIMIT=10
 CONFIG_LFS_FORMAT_IF_MOUNT_FAILED=n
-CONFIG_ASSERT_VERBOSE=y
+CONFIG_ASSERT_VERBOSE=n
 CONFIG_PINCTRL=y
 CONFIG_PINCTRL_HL_GAUDI3=y
